roles/vendors/hpe/tasks/iso.yml

```diff
@@ -7,9 +7,6 @@
     username: "{{ bmc_user }}"
     password: "{{ bmc_password }}"
 
```

Comment:
Do we really want to remove the eject in here? Wondering if we want to query (if possible) to figure out what URL to request to eject first, wdyt?